docs/flink-queries.md

```diff
@@ -80,6 +80,21 @@ Here are the SQL settings for the [FLIP-27](https://cwiki.apache.org/confluence/
 SET table.exec.iceberg.use-flip27-source = true;
 ```
 
+### Reading branches and tags with SQL
+Branch and tags can be read via SQL by specifying options. For more details
+refer to [Flink Configuration](../flink-configuration/#read-options)
+
+```sql
+--- Read from branch b1
+SELECT * FROM table OPTIONS('branch'='b1')
```

Comment:
You may have left out ` /*+ ……  */`:

`SELECT * FROM sample /*+ OPTIONS('branch'='b1')*/ ;`

Reply:
Oh yes, good catch!